modules/security/src/main/java/org/apache/ignite/internal/security/authentication/basic/BasicAuthenticator.java

```diff
@@ -26,23 +27,19 @@
 
 /** Implementation of basic authenticator. */
 public class BasicAuthenticator implements Authenticator {
-    private final String authenticatorName;
+    private final String providerName;
 
-    private final String username;
-
-    private final String password;
+    private final List<BasicUser> users;
```

Comment:
I think it should be `Map<String, BasicUser>` where the key is a username. WDYT?